README.md

```diff
@@ -16,6 +16,12 @@ brew tap icalialabs/formulae
 brew install plis
 ```
 
+On Arch Linux, you can install it from AUR:
+
+```
+yaourt -S plis
+```
+
```

Comment:
How it it that it's installed using `yaourt`? Who's keeping the AUR repository up to date?